### ISSUE TITLE
Move some Debug output to Trace

### ISSUE THREAD

**Issue by FG-TUM** [AutoPas/AutoPas#540]
**Is your feature request related to a problem? Please describe.**
Debug output is getting out of hand. This can slow down the simulation unnecessarily and produces output files that are way too large especially in MPI parallel simulations.

**Describe the solution you'd like**
Some Stuff that is only rarely used or only used for plotting should be moved to trace.

**Describe alternatives you've considered**
Introduce additional log levels for plotting?

~**Additional context**~

**Comment by FG-TUM** [comment]
After adding loggers for plot stuff this should be fine. This was done in #567 . We should still add a logger for load balancers of sliced traversals.

**Comment by FG-TUM** [comment]
Alternative idea: leave `Debug` as it is and use `Info` significantly more restrictive. Mainly for printing the current config per `iteratePairwise()`.
@SamNewcome your opinion?

**Comment by SamNewcome** [comment]
Sounds like a good idea for the future

**Comment by FG-TUM** [comment]
Yes but it is an everlasting and ongoing thing so we will not keep the issue open for this but rather tackle it as it becomes problematic.
